master/getting-started/kubernetes/installation/calico.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Installing Calico on Kubernetes with Calico networking
+canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/calico'
+---
+
+## Before you begin
+
+- Ensure that you have a Kubernetes cluster that meets the 
+  {{site.prodname}} [system requirements](../../requirements). If you don't, 
+  follow the steps in [Using kubeadm to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/) to create one.
+
+- If your cluster has RBAC enabled, issue the following command to 
+  configure the roles and bindings that {{site.prodname}} requires.
+
+   ```
+   kubectl apply -f \
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
+   ```
+   > **Note**: You can also 
+   > [view the manifest in your browser](hosted/rbac-kdd.yaml){:target="_blank"}.
+   {: .alert .alert-info}
+   
+## Installing {{site.prodname}} with {{site.prodname}} networking
+
+### About installing {{site.prodname}} with {{site.prodname}} networking
+
+The procedure differs according to the type of datastore you want {{site.prodname}}
+to use and the number of nodes. Refer to the section that matches your configuration.
+
+- [etcd datastore](#etcd-datastore)
+
+- [Kubernetes API datastore with 50 nodes or less](#kubernetes-api-datastore-with-50-nodes-or-less) (beta)
+
+- [Kubernetes API datastore with more than 50 nodes](#kubernetes-api-datastore-with-more-than-50-nodes) (beta)
+
+> **Note**: {{site.prodname}} networking with the Kubernetes API datastore is 
+> in beta due to the following.
+> - It does not yet support {{site.prodname}} IPAM. We recommend using 
+>   `host-local` IPAM in conjunction with Kubernetes pod CIDR assignments,
+>   as specified in our manifests.
+> - You must use `calicoctl` to configure the node-to-node mesh, default 
+>  autonomous system (AS) number, and all BGP peering.
+{: .alert .alert-info}
+
+### etcd datastore
+
+1. Open [the {{site.prodname}} networking manifest for etcd](hosted/calico.yaml){:target="_blank"},
+   copy its contents, and paste them into a new file with the same name.
+   
+1. In the `ConfigMap` named `calico-config`, set the value of 
+   `etcd_endpoints` to the IP address and port of your etcd server.
+   
+   > **Tip**: You can specify more than one using commas as delimiters.
+   {: .alert .alert-success}
+   
+1. Apply the manifest using the following command.
+
+   ```bash
+   kubectl apply -f calico.yaml
+   ```
+
+### Kubernetes API datastore with 50 nodes or less
+
+1. Ensure that the Kubernetes controller manager has the following flags
+   set: <br>
+   `--cluster-cidr=192.168.0.0/16` and `--allocate-node-cidrs=true`. 
+
+   > **Tip**: On kubeadm, you can pass `--pod-network-cidr=192.168.0.0/16` 
+   > to kubeadm to set both Kubernetes controller flags.
+   {: .alert .alert-success}
+   
+1. Issue the following command to install {{site.prodname}}.
+
+   ```bash
+   kubectl apply -f \
```

Comment:
I recommend dropping the newline here and putting this all on one line.

The rendered command still goes off the screen, and it makes it harder to copy.

Reply:
It's considered a best practice to add the breaks in bash sample code. While there are many people, some say too many, who will copy and paste anything, there are also those who want to see what is this code before they will decide to copy and paste it. There are also those who are simply reading and understanding and doing nothing. For all of these reasons, it's a better practice to add the breaks so people can also read the sample code (not just copy paste it).

Reply:
It's not code though, and you still can't see the full command with the line break so it's not solving that problem....

Reply:
The paths are rather convoluted but perhaps that is a discussion for another time.